data/data-pipeline/data_pipeline/etl/score/tests/test_score_post.py

```diff
@@ -112,27 +111,27 @@ def test_load_score_csv(etl, score_data_expected):
 
 def test_load_tile_csv(etl, tile_data_expected):
     reload(constants)
-    etl._load_score_csv(
+    etl._load_score_csv_full(
         tile_data_expected, constants.DATA_SCORE_CSV_TILES_FILE_PATH
     )
     assert constants.DATA_SCORE_CSV_TILES_FILE_PATH.is_file()
 
 
-def test_load_downloadable_zip(etl, monkeypatch, downloadable_data_expected):
-    reload(constants)
-    STATIC_FILES_PATH = (
-        Path.cwd() / "data_pipeline" / "files"
-    )  # need to monkeypatch to real dir
-    monkeypatch.setattr(constants, "FILES_PATH", STATIC_FILES_PATH)
-    monkeypatch.setattr(
-        constants,
-        "SCORE_DOWNLOADABLE_PDF_FILE_PATH",
-        STATIC_FILES_PATH / constants.SCORE_DOWNLOADABLE_PDF_FILE_NAME,
-    )
-    etl._load_downloadable_zip(
-        downloadable_data_expected, constants.SCORE_DOWNLOADABLE_DIR
-    )
-    assert constants.SCORE_DOWNLOADABLE_DIR.is_dir()
-    assert constants.SCORE_DOWNLOADABLE_CSV_FILE_PATH.is_file()
-    assert constants.SCORE_DOWNLOADABLE_EXCEL_FILE_PATH.is_file()
-    assert constants.SCORE_DOWNLOADABLE_ZIP_FILE_PATH.is_file()
+# def test_load_downloadable_zip(etl, monkeypatch, downloadable_data_expected):
```

Comment:
same question

Reply:
See [comment here](https://github.com/usds/justice40-tool/pull/1252#pullrequestreview-886465283).

Maybe instead of commenting out, we should just take these tests out.